protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -298,3 +298,17 @@ enum CoreOperatorType {
   NESTED_LOOP_JOIN = 35;
   AVRO_SUB_SCAN = 36;
 }
+
+message Func {
+  optional string name = 1;
+  repeated common.MajorType major_type = 2;
+}
+
+message Jar {
```

Comment:
Jar can contain one or more functions. The name is file name (example: DrillUDF-1.0.jar).
User doesn't have to ensure that name is unique between static and dynamic UDFs, it should be unique only between dynamic UDFs.